README.md

```diff
@@ -9,17 +9,30 @@ This is the Glitch equivalent of running `create-probot-app` to generate a new p
 
 To get your own Glitch-hosted Probot up-and-running:
 
-1. Head on back to our [Devlopment Docs](https://probot.github.io/docs/development/#configuring-a-github-app) to configure your GitHub App.
 
-2. Click the **New File** button (at left on Glitch) and type `.data/private-key.pem`. Then click **Add File**. Open the private key you downloaded during your Github App creation, and copy/paste the contents into your new file.
+1. [Configure a new app on Github](https://github.com/settings/apps/new).
+    - Hit the "Show" button on the top left of this page to find the URL. It will look something like `https://random-word.glitch.me/probot`, except the domain will be specific to your app.
+    - For the Homepage URL, use your app URL that you just found (updating the domain to match yours): `https://random-word.glitch.me/probot`
+    - For the Webhook URL, use this URL (again, updating the domain to match yours): `https://random-word.glitch.me/`. Notice that we left off the `/probot`.
+    - For the Webhook Secret, open a terminal and run `openssl rand -base64 32`. Copy/paste the outputted value to the Webhook Secret box. Keep this handy
+    until Step 4.
+    - Choose the permissions you want to give your bot based on what you want to build (ex. issues bot, PR bot, hybrid).
```

Comment:
'bot' -> 'app'. Also since the glitch example adds a label to an issue, we should probably explicitly says add issues read and write permission and listen on issue events.

Reply:
Yes!